bin/deployables

```diff
@@ -570,27 +570,64 @@ function copy_lambda_to_s3 () {
 
     aws_configure
 
-    aws --profile "${DEPLOY_AWS_PROFILE}" \
-        s3 cp \
-        ${DEPLOY_LAMBDA_ZIP_FULLPATH} \
-        s3://${DEPLOY_LAMBDA_S3_BUCKET}/${DEPLOY_LAMBDA_FUNCTION_NAME}/archive/${DEPLOY_LAMBDA_FUNCTION_NAME}-${DEPLOY_SHA1}.zip
-
-    if [[ "$?" != "0" ]] ; then
-        echo "Error: Failed to copy lambda zip to S3 (${DEPLOY_SHA1})"
-        exit 1
-    fi
+    # aws --profile "${DEPLOY_AWS_PROFILE}" \
+    #     s3 cp \
+    #     ${DEPLOY_LAMBDA_ZIP_FULLPATH} \
+    #     s3://${DEPLOY_LAMBDA_S3_BUCKET}/${DEPLOY_LAMBDA_FUNCTION_NAME}/archive/${DEPLOY_LAMBDA_FUNCTION_NAME}-${DEPLOY_SHA1}.zip
+    #
+    # if [[ "$?" != "0" ]] ; then
+    #     echo "Error: Failed to copy lambda zip to S3 (${DEPLOY_SHA1})"
+    #     exit 1
+    # fi
 
-    if [ ! -z "${DEPLOY_LAMBDA_ZIP_VERSION}" ] ; then
+  #  if [ ! -z "${DEPLOY_LAMBDA_ZIP_VERSION}" ] ; then
       aws --profile "${DEPLOY_AWS_PROFILE}" \
           s3 cp \
           ${DEPLOY_LAMBDA_ZIP_FULLPATH} \
-          s3://${DEPLOY_LAMBDA_S3_BUCKET}/${DEPLOY_LAMBDA_FUNCTION_NAME}/${DEPLOY_LAMBDA_FUNCTION_NAME}-${DEPLOY_LAMBDA_ZIP_VERSION}.zip
+          s3://${DEPLOY_LAMBDA_S3_BUCKET}/${DEPLOY_LAMBDA_FUNCTION_NAME}/${DEPLOY_LAMBDA_FUNCTION_NAME}.zip
 
       if [[ "$?" != "0" ]] ; then
-          echo "Error: Failed to copy lambda zip to S3 (${DEPLOY_LAMBDA_ZIP_VERSION})"
+          echo "Error: Failed to copy lambda zip to S3 (${DEPLOY_LAMBDA_ZIP_FULLPATH})"
           exit 1
       fi
 
```

Comment:
@jdsipes let's talk about strategies here.   the idea behind deploying the DEPLOY_SHA1 based tag is to have a version to test based on the commit, and not just based on a release or an explicit version number.   

as for having a DEPLOY_LAMBDA_ZIP_VERSION based version, the idea is that it's easily identifable artifact for past tags/releases.

what are your thoughts on why those aren't needed?

Reply:
We could, but I think that we are using the S3 versioning that, I believe, is (or can be) set up to automatically archive. Maybe we can pass those types of values using the --metadata flag. One a side note, getting the github sha in circleci should be straight forward, I wasn't sure how to get the release version... maybe that we just have to manually update in the circle.yml file?

Reply:
but S3 versions have nothing to do with release versions, correct?   e.g. everytime you build the zip and upload it, S3 will give you a new version.    versus we want to tag something for general release and then have a copy of that zip at the ready without having to map into S3 versions.

am i missing something here?